.github/workflows/clean.yml

```diff
@@ -0,0 +1,38 @@
+name: Clean Github container registry
+on:
+  workflow_call: { }
+  pull_request:
+    types: [ closed ]
+
+jobs:
+  clean-registry:
+    env:
+      REGISTRY: ghcr.io
+      IMAGE_NAME: ${{ github.repository }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
it would be nice to extract matrix to a single file ind include it on all workflows, that require it, if its possible.